project.clj

```diff
@@ -13,6 +13,8 @@
                  [io.pedestal/pedestal.service "0.5.3"]
                  [io.pedestal/pedestal.service-tools "0.5.3"]
                  [io.pedestal/pedestal.jetty "0.5.3"]
+                 [slingshot "0.12.2"]
```

Comment:
Would like to avoid using slingshot, is a very old library, unmaintained, and is just a syntax sugar for try catch

Reply:
Ah, não sabia.. tava na documentação do clj-http e achei simples de tratar os erros http